src/components/collection/Config.tsx

```diff
@@ -56,29 +54,15 @@ function CollectionConfig() {
             >
                 <>
                     <CollectionSelector />
-                    <Grid
-                        container
-                        spacing={1}
-                        columns={{ xs: 4, sm: 8, md: 12, lg: 12 }}
-                    >
-                        {collections.map((collection: any, index: number) => {
-                            return (
-                                <Grid
-                                    item
-                                    key={`CollectionResourceConfig-${index}`}
-                                    xs={4}
-                                    sm={4}
-                                    md={4}
-                                    lg={3}
-                                >
-                                    <ExpandableResourceConfig
-                                        collectionName={collection}
-                                        id={imageTag.id}
-                                    />
-                                </Grid>
-                            );
-                        })}
-                    </Grid>
+                    {collections.map((collection: any, index: number) => {
+                        return (
+                            <ExpandableResourceConfig
+                                collectionName={collection}
+                                id={imageTag.id}
+                                key={`CollectionResourceConfig-${index}`}
+                            />
+                        );
+                    })}
```

Comment:
Putting stuff in grid caused some view to spill over the edge and become unusable.